client/src/components/_common/AppIcon/AppIcon.jsx

```diff
@@ -11,8 +11,14 @@ const AppIcon = ({ appId }) => {
     Object.keys(appIcons).forEach((appName) => {
```

Comment:
for the icons, it looks like we need more room for them the dashboard and history tables
![Screen Shot 2022-05-06 at 12 20 55 PM](https://user-images.githubusercontent.com/8287580/167182412-518f6f6f-9df8-4c7e-b825-244178837e9b.png)
![Screen Shot 2022-05-06 at 12 20 36 PM](https://user-images.githubusercontent.com/8287580/167182415-583f37c0-124d-4de8-a804-cabdf7018cc0.png)
:

Reply:
@nathanfranklin Good find. New ticket: [FP-1638](https://jira.tacc.utexas.edu/browse/FP-1638)